### ISSUE TITLE
Scroll to top only works on some screens

### ISSUE THREAD

**Issue by ijm8710** [lichess-org/mobile#978]
**DOES**

Scroll down on settings screen, click status bar and it does bring you to top

Scroll down on a variant page (ie your bullet stats) and it does bring you to top

Scroll down on your overall profile and it does bring you to top

Go to puzzles theme page and expand some categories so it’s more than one scroll page long; it does bring you to top

**DOES NOT (hence inconsistent)**

Scroll down on watch tab, click status bar and it does not bring you to top 

Scroll down on puzzles tab and click status bar and it does not bring you to top

Scroll down on home tab and click status bar and it does not bring you to top

But the biggest place is if you dive on filtered game archive, especially after fetching a bunch of new pages, it does not and heavily would expect it to bring you to top when clicking status bar

I’m pretty confident some (probably not all) of these tabs had click status bar to scroll to top implementation previously that were probably broken with conflicts

**Comment by tom-anders** [comment]
Note that this is ios-only behavior: https://docs.flutter.dev/platform-integration/platform-adaptations#return-to-top

And it seems like the flutter framework handles it automatically, so it's weird that it works only on some screens

**Comment by ijm8710** [comment]
Yeah. Most screens I don't care as much but screens where yall load pages is where it would be helpful like the game archive and puzzle history

**Comment by ijm8710** [comment]
Studies feed is another example of this

**Comment by ijm8710** [comment]
@veloce one last ticket to please get your opinion on (tried choosing 4 of mine that have sat for a bit). This one, it's also very noticeable on broadcast screen when I want to easily get back to the top after going far into completed events at bottom (this is also why I'd love to be able to click the broadcasts header "ongoing/completed" and have option to jump sections)

**Comment by ijm8710** [comment]
Recent games expanded no longer scrolls to top now as well
